master/master.cfg

```diff
@@ -261,6 +264,21 @@ test_factory.addStep(ShellCommand(
     logfiles={"console" : { "filename" : "console.log", "follow" : False }},
     decodeRC={0 : SUCCESS, 1 : FAILURE, 2 : WARNINGS, 3 : SKIPPED },
     description=["zfsstress"], descriptionDone=["zfsstress"]))
+test_factory.addStep(Git(repourl=zfs_fedora_installer_repo,
+    workdir="build/tests/zfs-fedora-installer",
+    mode="full", method="clobber", codebase="zfs-fedora-installer",
+    logEnviron=False, getDescription=True,
+    description=["cloning"], descriptionDone=["cloned"]))
```

Comment:
This isn't a bad way to go but it's probably more flexible to adopt the approach taken in [bb-test-xfstests.sh](https://github.com/zfsonlinux/zfs-buildbot/blob/master/scripts/bb-test-xfstests.sh) where the script just wget's a tarball from github.  This has a few advantages:

* It can be all done in a single build step and is largely self contained to one script.
* We can point it by default at a specific version or the head of your git repository.
* Setting TEST_FEDORA_INSTALLER_URL in your commit message can be used to override the default.